web-app/backend/app/config.py

```diff
@@ -2,5 +2,5 @@
 basedir = os.path.abspath(os.path.dirname(__file__))
 
 class Config(object):
-    SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
+    SQLALCHEMY_DATABASE_URI = "postgresql://localhost/JVP-Dev"
```

Comment:
Did you wanna change this back? If somebody else ever wanted to use the app there db would have to be called jvp-dev

Reply:
reverted